### PR TITLE
feat: LLM tool support graphic display

### PR DESCRIPTION
**Pull request by lixinghua123**

### CODE REVIEW

File: packages/lb-components/src/components/LLMToolView/index.tsx [hunk 1]
```diff
@@ -56,20 +56,32 @@ const LLMToolView: React.FC<IProps> = (props) => {
...
+    }
+    if (LLMConfig?.dataType?.response === ELLMDataType.Text) {
+      if (newAnswerList.length > 0) {
+        list = newAnswerList;
+      }
     }
     setAnswerList(list);
     setModelAPIResponse(currentResult?.modelAPIResponse || []);
-  }, [imgIndex, newAnswerList]);
+  }, [imgIndex, newAnswerList, LLMConfig]);
```
Comment (Glenfiddish) [inline packages/lb-components/src/components/LLMToolView/index.tsx:84]: observer newAnswerList is needed?

File: packages/lb-components/src/components/LLMToolView/sidebar/components/textEditor/index.tsx [hunk 2]
```diff
@@ -48,12 +48,18 @@ const TextEditor = (props: IProps) => {
       }}
       style={{ marginBottom: '16px' }}
     >
-      <div style={{ marginBottom: '16px' }}>
+      <Form.Item
+        name='title'
+        style={{ marginBottom: '16px' }}
+        label={' '}
```
Comment (Glenfiddish) [inline packages/lb-components/src/components/LLMToolView/sidebar/components/textEditor/index.tsx:54]: why ' ', not ''

File: packages/lb-components/src/components/LLMToolView/questionView/components/imgView/index.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  const { answerList, hoverKey } = props;
+  const ImgFail = i18n.language === 'en' ? ImgFailEn : ImgFailCn;
+  return (
+    <div style={{ display: 'flex', flexWrap: 'wrap' }}>
+      {answerList.map((i: IAnswerList, index: number) => {
+        return (
+          <div
+            key={index}
+            style={{
+              margin: '0px 40px 40px 0px',
```
Comment (Glenfiddish) [inline packages/lb-components/src/components/LLMToolView/questionView/components/imgView/index.tsx:31]: use css class not inline~

File: packages/lb-components/src/components/LLMToolView/types.ts [hunk 4]
```diff
@@ -20,6 +20,7 @@ export interface IAnswerList {
     [key: string]: boolean;
   };
   newAnswer?: string;
+  url?: string;
```
Comment (Glenfiddish) [inline packages/lb-components/src/components/LLMToolView/types.ts:23]: useage for url

File: packages/lb-components/src/components/LLMToolView/sidebar/components/textEditor/index.tsx (hunk 2 above)
Comment (lixinghua123) [reply packages/lb-components/src/components/LLMToolView/sidebar/components/textEditor/index.tsx:54]: Required to display "*"

File: packages/lb-components/src/components/LLMToolView/index.tsx (hunk 1 above)
Comment (lixinghua123) [reply packages/lb-components/src/components/LLMToolView/index.tsx:84]: need

**Pull request merged**